### DISCUSSION TITLE
Resource Directory Error on Startup

### DISCUSSION THREAD

**Discussion by pmaine** [AlexandreRouma/SDRPlusPlus#283] [Q&A] [answered]
I appreciate everything the developers are doing but their is room for improvement in the documentation for building from source. Please note the following error and suggest a solution.

pi@raspberrypi:~/source/SDRPlusPlus-1.0.3/build $ ./sdrpp
[2021-08-20 08:26:17.036] [info] SDR++ v1.0.3
[2021-08-20 08:26:17.036] [info] Loading config
[2021-08-20 08:26:17.037] [warning] ConfigManager locked, waiting...
[2021-08-20 08:26:17.200] [error] Resource directory doesn't exist! Please make sure that you've configured it correctly in config.json (check readme for details)

**Comment by starvald** [comment]
https://github.com/AlexandreRouma/SDRPlusPlus#running-for-development

**Comment by AlexandreRouma** [comment, marked as answer]
Hello,

This is clearly described in [this section of the readme](https://github.com/AlexandreRouma/SDRPlusPlus#running-for-development-1).

If you're not trying to develop plugins for SDR++, please just run `sudo make install` and use it like any other software.

**Reply by pmaine** [reply]
I am not trying to build for development, I am just trying to build for a Raspberry Pi 4B from source code.
What do I need to do next?

**Reply by starvald** [reply]
just sudo make install, it will install the resources/modules where they need to go.

**Reply by pmaine** [reply]
@starvald your last comment worked perfectly for me!

**Reply by pmaine** [reply]
@starvald I have tried to mark your last response as an answer but github doesn't let me.
